packages/i18n/src/locales/en.i18n.json

```
@@ -6405,8 +6405,8 @@
   "cloud.RegisterWorkspace_Setup_Terms_Privacy": "I agree with <1>Terms and Conditions</1> and <3>Privacy Policy</3>",
   "Larger_amounts_of_active_connections": "For larger amounts of active connections you can consider our <1>multiple instance solutions</1>.",
   "Uninstall_grandfathered_app": "Uninstall {{appName}}?",
-  "App_will_lose_grandfathered_status": "**This app will lose its grandfathered status.**  \n  \nWorkspaces on Community can have up to {{limit}} apps enabled. Grandfathered apps count towards the limit but the limit is not applied to them.",
-  "App_will_lose_grandfathered_status_private": "**This app will lose its grandfathered status.**  \n  \nBecause Community workspaces cannot enable private apps, this workspace will require a premium plan in order to enable this app again in future.",
+  "App_will_lose_grandfathered_status": "**This app will lose its app limit policy exemption.**  \n  \nWorkspaces on Community can have up to {{limit}} apps enabled. Uninstalling this app will cause it to lose its exemption policy.",
+  "App_will_lose_grandfathered_status_private": " **This app will lose its app limit policy exemption.**  \n  \nBecause Community workspaces cannot enable private apps, this workspace will require a premium plan in order to enable this app again in future.",
```

```suggestion
  "App_will_lose_grandfathered_status_private": "**This app will lose its app limit policy exemption.**  \n  \nBecause Community workspaces cannot enable private apps, this workspace will require a premium plan in order to enable this app again in future.",
```
I think we shouldn't have an extra leading space here, right?